resources/services/kinesis/gen.hcl

```diff
@@ -1,7 +1,10 @@
+service          = "aws"
```

Comment:
Could we add `//check-for-changes` here? Technically you don't have to because it's not a new file, but it would be nice 😄